var/spack/repos/builtin/packages/py-scikit-build-core/package.py

```diff
@@ -19,6 +19,7 @@ class PyScikitBuildCore(PythonPackage):
 
     license("Apache-2.0")
 
+    version("0.10.7", sha256="20e3174dbd788928319d452f803c374ba046ba2194ea992a1ca964c8f7ab221c")
```

Comment:
I can't confirm this checksum.
```
$ curl -L https://files.pythonhosted.org/packages/source/s/scikit_build_core/scikit_build_core-0.10.7.tar.gz | sha256sum 
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  249k  100  249k    0     0   961k      0 --:--:-- --:--:-- --:--:--  961k
04cbb59fe795202a7eeede1849112ee9dcbf3469feebd9b8b36aa541336ac4f8  -
```
So, based on this, I would suggest
```suggestion
    version("0.10.7", sha256="04cbb59fe795202a7eeede1849112ee9dcbf3469feebd9b8b36aa541336ac4f8")
```
Can you verify how you got the checksum here?

Reply:
I did
```bash
$ curl -L https://github.com/scikit-build/scikit-build-core/archive/refs/tags/v0.10.7.tar.gz   | sha256sum 
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  256k    0  256k    0     0   662k      0 --:--:-- --:--:-- --:--:--  662k
20e3174dbd788928319d452f803c374ba046ba2194ea992a1ca964c8f7ab221c  -
```
What should be the 'reference' tarball source for Spack?

Reply:
When a python package has a `pypi` attribute, the reference is pypi.